### PR TITLE
Clean Up Painless Cast Object

### PR DESCRIPTION
**Pull request by jdconrad**
Adds five static methods to the Cast object to make each type of cast easier to see within the code for unboxing/boxing.

### CODE REVIEW

File: modules/lang-painless/src/main/java/org/elasticsearch/painless/Definition.java [hunk 1]
```diff
@@ -384,6 +384,26 @@ public Method getFunctionalMethod() {
     }
 
     public static class Cast {
+        public static Cast standard(Type from, Type to, boolean explicit) {
```
Comment (rjernst) [inline modules/lang-painless/src/main/java/org/elasticsearch/painless/Definition.java]: If we're going to have these helper methods, please at least give a one line javadoc explaining each one.

**Review by rjernst — APPROVED** [review]
This looks fine as it is purely mechanical. I have one request.

**Comment by jdconrad** [comment]
@elasticmachine test this please

File: modules/lang-painless/src/main/java/org/elasticsearch/painless/Definition.java (hunk 1 above)
Comment (jdconrad) [reply modules/lang-painless/src/main/java/org/elasticsearch/painless/Definition.java]: Done.

**Comment by jdconrad** [comment]
@rjernst Thanks for the review.

**Pull request merged**